src/OrchardCore.Modules/OrchardCore.Lists/Services/ContainerService.cs

```diff
@@ -283,6 +290,29 @@ public async Task<IEnumerable<ContentItem>> QueryContainedItemsAsync(string cont
                     return containedItems;
                 }
 
+                if (listPartFilter.DisplayText != null)
+                {
+                    containedItems = containedItems.Where<ContentItem>(i => i.ContentItem.As<TitlePart>().Title.Contains(listPartFilter.DisplayText)).ToList();
+                }
+                if ((int)listPartFilter.Status != (int)ListPartFilterViewModel.ContentsStatus.None)
+                {
+                    switch ((int)listPartFilter.Status)
+                    {
+                        case (int)ListPartFilterViewModel.ContentsStatus.Draft:
+                            containedItems = containedItems.Where(i => i.ContentItem.HasDraft());
+                            break;
+                        case (int)ListPartFilterViewModel.ContentsStatus.Published:
+                            containedItems = containedItems.Where(i => i.ContentItem.Published);
+                            break;
+                        case (int)ListPartFilterViewModel.ContentsStatus.Owner:
+                            containedItems = containedItems.Where(i => i.Owner == _hca.HttpContext.User.Identity.Name);
```

Comment:
Gondolj arra, hogy itt a `HttpContext` és utána is minden lehet `null`.

Reply:
Továbbra is.

Reply:
Csak úgy sikerült megoldani hogy felette if-ben megvizsgáltam minden szükséges paramétert egyenként.

Reply:
Már megoldva szebben

Reply:
Ez továbbra is probléma. A `HttpContext` **és utána is minden** lehet `null`. Most ha nincs bejelentkezett user, akkor `NullReferenceExpression` lesz. Gondolj arra, hogy ezt a service-t nem csak az az admin lista használhatja.

Reply:
Így már jó lehet?
```
string loggedUser = _hca?.HttpContext?.User?.Identity?.Name;
if (loggedUser != null)
 {
     containedItems?.With<ContentItemIndex>().Where(i => i.Owner == loggedUser);
}
```

Reply:
A `_hca` nem lehet `null` (hacsak nem megy valami nagyon félre, de akkor legyen exception). Amúgy ja, bár inább `currentUserName` vagy ilyesmi.